src/elements/e-video.vue

```diff
@@ -97,108 +85,126 @@
 
       /**
        * Allows to set the video player width.
+       * width can either be responsive or hard coded.
+       * youtube standard -> width="560".
        */
       width: {
-        type: [String, Number],
-        default: 560,
+        type: String,
+        default: null,
       },
 
       /**
        * Allows to set the video player height.
+       * height can either be responsive or hard coded.
+       * youtube standard -> height="315".
        */
       height: {
-        type: [String, Number],
-        default: 315,
-      },
-
-      /**
-       * Defines the players size to be responsive 100% width to the parent container
-       */
-      responsive: {
-        type: Boolean,
-        default: false,
+        type: String,
+        default: null,
       },
     },
 
     // emits: {},
     // setup(): Setup {},
 
     computed: {
+      /**
+       * Returns a validity check for the video url.
```

Comment:
todo: this computed does not `return a validity check` (sound like it would return a custom validation method as return value). It checks, if we have a valid url.